test-projects/demo/dashboards/welcome.visivo.yml

```diff
@@ -71,9 +71,14 @@ dashboards:
           - width: 2
             table:
               name: Fibonacci Waterfall Table
-              trace: ref(Fibonacci Waterfall Trace)
+              traces:
+                - ref(Fibonacci Waterfall Trace)
               columns:
-                - header: Widget Type
-                  column: x_data
-                - header: Widget Sales
-                  column: y_data
+                - trace_name: Fibonacci Waterfall Trace
+                  column_defs:
+                    - header: Widget Type
+                      key: columns.x_data
+                    - header: Widget Sales
+                      key: props.y
```

Comment:
I am thinking this way of referencing could make it easier to use and you would have to add column to the trace.  Thoughts?

Reply:
Yeah I like this better.